spec/unit/crypto/algorithms/megolm.spec.js

```diff
@@ -540,6 +540,12 @@ describe("MegolmDecryption", function() {
             content: {},
         });
         await aliceClient._crypto.encryptEvent(event, aliceRoom);
+        await new Promise((resolve) => {
+            // encryptMessage retries senders in the background before giving
+            // up and telling them that there's no olm channel, so we need to
+            // wait a bit before checking that we got the message
+            setTimeout(resolve, 100);
```

Comment:
Do we use the `jest.useFakeTImers()` API?
https://jestjs.io/docs/en/timer-mocks

Reply:
I think we have some things that use jest timer mocks, but I'm not sure how to use them in the context of waiting for a background task to complete.  Though looking at it again, I see a way of doing it without the timer at all.